oz/redis_sessions/middleware.py

```diff
@@ -24,7 +24,17 @@ def _session_key(self):
 
             if not session_id:
                 session_id = oz.redis_sessions.random_hex(20)
-                self.set_secure_cookie("session_id", session_id.encode('utf-8'), httponly=True)
+                if "cookie_domain" in oz.settings:
```

Comment:
If you check the tornado source code, it is indeed equivalent. `set_secure_cookie` passes the kwarg to `set_cookie`, whose default value is `None`.